app/models/veteran.rb

```diff
@@ -13,10 +13,17 @@
 #
 
 class Veteran < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
   EMAIL_REGEX = /\A[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]+\z/
```

Comment:
I'd recommend reading up on [e-mail regexes](http://www.regular-expressions.info/email.html) for the lolz. There are a few variations of official RFC 5322 e-mail addresses that this regex does not support.